backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlConfig.java

```diff
@@ -47,7 +46,10 @@ public DataFetcherExceptionResolver dataFetcherExceptionResolver() {
       log.error("Will replace the following exception with a GenericGraphqlException: ", exception);
 
       if (exception instanceof ConflictingUserException) {
-        return Mono.just(singletonList((GraphQLError) exception));
+        String errorBody =
```

Comment:
Hopefully, cleaning this up addresses: https://github.com/CDCgov/prime-simplereport/pull/5165#discussion_r1093797923